components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/ConfigurationConstants.java

```diff
@@ -102,7 +107,13 @@
         ERROR_CODE_REPLACE_RESOURCE("CONFIGM_00034", "Error while replacing the resource : %s."),
         ERROR_CODE_REPLACE_ATTRIBUTE("CONFIGM_00035", "Error while replacing the attribute: %s."),
         ERROR_CODE_FEATURE_NOT_ENABLED("CONFIGM_00036", "Configuration management feature is not enabled."),
-        ;
+        ERROR_CODE_FILE_IDENTIFIERS_REQUIRED("CONFIGM_00036", "File identifier validation failed for " +
```

Comment:
Each of the added error codes should be in ascending order.

Reply:
done